### PR TITLE
css-media-queries - Cindy Le

### PR DESCRIPTION
**Pull request by cindy-le91**
Kapture 2023-04-05 at 20 47 35
Kapture 2023-04-05 at 20 49 06
Kapture 2023-04-05 at 20 53 13

### CODE REVIEW

**Review by robgmerrill — APPROVED** [review]
👍

File: css-media-queries/css-media-queries-quiz-notes.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# css-media-queries-quiz-notes
+
+## Quiz Questions
+
+After completing this exercise, you should be able to discuss or answer the following questions:
+
+- Give two examples of media features that you can query in an `@media` rule.
+any-hover
```
Comment (robgmerrill) [inline css-media-queries/css-media-queries-quiz-notes.md:8]: Please revisit this question :)

**Review by robgmerrill — CHANGES_REQUESTED** [review]
One quiz question to revisit :)

**Comment by cindy-le91** [comment]
Screen Shot 2023-04-06 at 9 46 03 AM

**Review by robgmerrill — APPROVED** [review]
👍

**Pull request merged**